src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -515,6 +516,23 @@ public Task SendFunctionEnvironmentReloadRequest()
             return _reloadTask.Task;
         }
 
+        public Task SendWorkerWarmupRequest()
+        {
+            _workerChannelLogger.LogDebug("Sending WorkerWarmupRequest to WorkerProcess with Pid: '{0}'", _rpcWorkerProcess.Id);
+
+            WorkerWarmupRequest request = new WorkerWarmupRequest()
```

Comment:
nit: we don't need to repeat the type 

```suggestion
            var request = new WorkerWarmupRequest()
```

or

```suggestion
            WorkerWarmupRequest request = new()
```